### PR TITLE
Rudy feature dbqueries

### PR DESCRIPTION
**Pull request by rudyesar**
includes some work on styling, dbqueries, and modifying code to pass more data to checkout

### CODE REVIEW

File: server/database/schema.sql [hunk 1]
```diff
@@ -125,21 +125,21 @@ ALTER TABLE "messages" ADD CONSTRAINT "messages_fk0" FOREIGN KEY ("thread_id") R
...
-  --       set latlng = case
-  --           when item_id > 1 then replace(latlng, '|',',"lng":')
-  --           else latlng
-  --         end;
-  --    update items
-  --         set latlng = case
-  --             when item_id >= 0 then replace(latlng, latlng,`{"lat":${latlng}}`)
-  --                 else latlng
-  --             end;
+  update items
```
Comment (Chloe-Meinshausen) [inline server/database/schema.sql]: Hey can you comment these out again?

Comment (rudyesar) [reply server/database/schema.sql]: I'll do that right now

File: client/src/itemView/CarouselContainer.jsx [hunk 2]
```diff
@@ -39,3 +46,31 @@ const CarouselContainer = (props) => {
 export default CarouselContainer;
 
 
+// const CarouselContainer = (props) => {
```
Comment (Chloe-Meinshausen) [inline client/src/itemView/CarouselContainer.jsx:49]: Is this code you are going to use again?

**Review by Chloe-Meinshausen — APPROVED** [review]
Hey over all it looks great! good work!

**Pull request merged**